File: TEST data for large repo/repo_Copy-2/Blueprism/BluePrism.Server.Domain.Models/ScheduleLogParameters.vb
Imports System.Runtime.Serialization
Imports BluePrism.Server.Domain.Models.DataFilters
Imports BluePrism.Server.Domain.Models.Pagination
Imports Func

<Serializable()>
<DataContract([Namespace]:="bp")>
Public Class ScheduleLogParameters
    <IgnoreDataMember>
    Public Property ScheduleId As DataFilter(Of Integer)
    <IgnoreDataMember>
    Public Property ItemsPerPage As Integer
    <IgnoreDataMember>
    Public Property StartTime As DataFilter(Of DateTimeOffset)
    <IgnoreDataMember>
    Public Property EndTime As DataFilter(Of DateTimeOffset)
    <IgnoreDataMember>
    Public Property ScheduleLogStatus As DataFilter(Of ItemStatus)
    <IgnoreDataMember>
    Public Property PagingToken As [Option](Of ScheduleLogsPagingToken)

End Class

File: TEST data for large repo/repo_Copy-2/Blueprism/BluePrism.Api/BluePrism.Api.Models/ScheduleLogParameters.cs
namespace BluePrism.Api.Models
{
    using System;

    public class ScheduleLogParameters : IPagingModel<int>
    {
        public int? ItemsPerPage { get; set; }
        public CommaDelimitedCollection<ScheduleLogStatus> ScheduleLogStatus { get; set; }
        public RangeFilterModel<DateTimeOffset?> StartTime { get; set; }
        public RangeFilterModel<DateTimeOffset?> EndTime { get; set; }
        public PagingTokenModel<int> PagingToken { get; set; }
    }
}
